### PR TITLE
Implement the missing DataFrame.swaplevel 

### PR DESCRIPTION
**Pull request by xinrong-databricks**

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -6355,6 +6355,159 @@ def sort_index(
...
+        1        6    6
+
+        >>> kdf.swaplevel('number', 'color', axis=1)
+        number   1    2
+        color  red blue
+        size     s    m
+        0        5    5
+        1        6    6
+        """
+        if axis == 0 or axis == "index":
```
Comment (ueshin) [inline databricks/koalas/frame.py:6450]: Shall we use `validate_axis`?

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -6355,6 +6355,159 @@ def sort_index(
...
+        Returns
+        -------
+        DataFrame
+            DataFrame with levels swapped in MultiIndex.
+
+        Examples
+        --------
+        >>> midx = pd.MultiIndex.from_arrays(
+            [['red', 'blue'], [1, 2], ['s', 'm']], names = ['color', 'number', 'size'])
+        >>> midx  # # doctest: +SKIP
```
Comment (ueshin) [inline databricks/koalas/frame.py:6379]: nit: remove a redundant `#`.

**Review by ueshin — COMMENTED** [review]
Btw, I guess we can rewrite `Series.swaplevel` with `DataFrame.swaplevel` now?

File: databricks/koalas/frame.py (hunk 2 above)
Comment (xinrong-databricks) [reply databricks/koalas/frame.py:6379]: Modified!

File: databricks/koalas/frame.py (hunk 1 above)
Comment (xinrong-databricks) [reply databricks/koalas/frame.py:6450]: Makes sense! Modified.

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -6355,6 +6355,160 @@ def sort_index(
...
+
+        Returns
+        -------
+        DataFrame
+            DataFrame with levels swapped in MultiIndex.
+
+        Examples
+        --------
+        >>> midx = pd.MultiIndex.from_arrays(
+        ... [['red', 'blue'], [1, 2], ['s', 'm']], names = ['color', 'number', 'size'])
```
Comment (ueshin) [inline databricks/koalas/frame.py:6378]: nit: Could you add an indent?

```py
>>> midx = ...
...     [['red', ...
```

Comment (xinrong-databricks) [reply databricks/koalas/frame.py]: Sounds good!

File: databricks/koalas/series.py [hunk 4]
```diff
@@ -1270,7 +1270,7 @@ def reset_index(
         else:
             return kdf
 
-    def to_frame(self, name: Union[Any, Tuple] = None) -> SparkDataFrame:
+    def to_frame(self, name: Union[Any, Tuple] = None) -> DataFrame:
```
Comment (xinrong-databricks) [inline databricks/koalas/series.py:1273]: Correcting return type.

Comment (ueshin) [reply databricks/koalas/series.py:1273]: Nice!

**Review by ueshin — APPROVED** [review]
LGTM, pending tests.

**Comment by ueshin** [comment]
Thanks! merging.

**Comment by itholic** [comment]
Cool !! Nice

**Pull request merged**